src/code-examples/dialog/html.js

```diff
@@ -1,3 +1,19 @@
-const html = 'dialog HTML';
+const html = `<button class="js-open-modal" aria-expanded="true" aria-controls="dialog-content">Open a Dialog</button>
+
+<div id="dialog-content" class="dialog" aria-hidden="false" role="dialog" aria-labelledBy="h2" aria-modal="true">
+  <button class="js-close-modal" aria-label="close the dialog" id="close-dialog">x</button>
```

Comment:
I'd add the attribute `type="button"` to your button element. Zastrow mentioned that I should add this to my own component since the default type for buttons is 'submit', which can have some unintended consequences in forms.

Reply:
Ah I did not know this. This would be for all 3 buttons, yes? That's another great point- changed! https://github.com/sparkbox/accessible-components/pull/88/commits/8582d806b3fe1520dcc80aec9c69abd7b6c3e308